packages/storybook/stories/select.js

```diff
@@ -616,6 +622,20 @@ angular.module('app', [])
     );
   })
   .add('interactive', () => {
+    const options = [
+      { value: 'one', label: 'One' },
+      { value: 'two', label: 'Two' },
+    ];
+    let selectState = {
+      open: false,
+    };
+    function onChange(event) {
```

Comment:
I see the `onChange` handler called. Lines 633–636 are never called—that behavior comes in a later story. I'll remove it for now. Are you seeing something else?